### PR TITLE
Add CPU and RAM limits for Docker containers

### PR DESCRIPTION
**Pull request by robertjndw**

### CODE REVIEW

File: shared/utils/config.go [hunk 1]
```diff
@@ -32,3 +36,21 @@ func LoadConfig(cfg interface{}) {
 
 	log.Debug("Config loaded: ", cfg)
 }
+
+func ParseMemoryLimit(limit string) (int64, error) {
+	unit := limit[len(limit)-2:]
```
Comment (Mtze) [inline shared/utils/config.go:41]: I think this will fail if `G` instead of `GB` is used, right?

File: HadesScheduler/docker/docker.go [hunk 2]
```diff
@@ -27,6 +27,8 @@ type DockerConfig struct {
 	DockerHost           string `env:"DOCKER_HOST" envDefault:"unix:///var/run/docker.sock"`
 	ContainerAutoremove  bool   `env:"CONTAINER_AUTOREMOVE" envDefault:"true"`
 	DockerScriptExecutor string `env:"DOCKER_SCRIPT_EXECUTOR" envDefault:"/bin/bash -c"`
+	CPU_limit            uint   `env:"DOCKER_CPU_LIMIT"`
+	RAM_limit            string `env:"DOCKER_RAM_LIMIT"`
```
Comment (Mtze) [inline HadesScheduler/docker/docker.go:31]: I guess it is a good idea to add a comment with the intended format.
```suggestion
	CPU_limit            uint   `env:"DOCKER_CPU_LIMIT"` // Number of CPUs - e.g. '6'
	RAM_limit            string `env:"DOCKER_RAM_LIMIT"` // RAM usage in GB or MB  - e.g. '4GB' 
```

File: shared/utils/config.go (hunk 1 above)
Comment (robertjndw) [reply shared/utils/config.go]: Fixed in the latest version

File: HadesScheduler/docker/docker.go (hunk 2 above)
Comment (robertjndw) [reply HadesScheduler/docker/docker.go]: I've changed the logic and also the naming. Not it supports the memory limit similarly to Docker itself (e.g. `4g` and `4m`)

**Review by Mtze — APPROVED** [review]

**Pull request merged**